tests/nhc-operator/tests/nhc_helpers.go

```
@@ -605,6 +606,148 @@ func buildNHCWithTestRemediation(name string) *unstructured.Unstructured {
+		tmpl["namespace"] = remediator.namespace
+	}
+
+	return map[string]interface{}{
+		"remediationTemplate": tmpl,
+		"order":               order,
+		"timeout":             timeout,
+	}
+}
+
+// validEscalationStepRaw returns a valid raw escalation step for SNR.
+func validEscalationStepRaw(order int64, timeout string) map[string]interface{} {
```

golangci-lint (2.12.2)

[error] validEscalationStepRaw - order always receives 0

(unparam)